x/rollapp/types/rollapp.go

```diff
@@ -1,25 +1,53 @@
 package types
 
 import (
+	"encoding/base64"
+	"fmt"
+	"net/url"
+	"regexp"
+	"strings"
+
 	errorsmod "cosmossdk.io/errors"
 	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	"github.com/dymensionxyz/dymension/v3/testutil/sample"
 )
 
-func NewRollapp(creator string, rollappId string, maxSequencers uint64, permissionedAddresses []string, transfersEnabled bool) Rollapp {
-	ret := Rollapp{
-		RollappId:             rollappId,
-		Creator:               creator,
-		MaxSequencers:         maxSequencers,
-		PermissionedAddresses: permissionedAddresses,
+func NewRollapp(
+	creator,
+	rollappId,
+	initSequencerAddress,
+	bech32Prefix,
+	genesisChecksum,
+	alias string,
+	metadata *RollappMetadata,
+	transfersEnabled bool,
+) Rollapp {
+	return Rollapp{
+		RollappId:               rollappId,
+		Creator:                 creator,
+		InitialSequencerAddress: initSequencerAddress,
```

Comment:
not sure where `InitialSequencerAddress` is used.
also not sure we need to keep it in the `x/rollapp`. why not creating it on the `x/sequencers` module?

Reply:
https://www.notion.so/dymension/ADR-x-Rollapp-registration-process-revision-1-07f3d4f2ceed4ae9ab44f609abb6b1d1?pvs=4#d28f2e5ec7d141bc8bd6e73d27937a7d

Reply:
IMO it is not supposed to be in the `rollapp` object.
this feature is an enforcement on the initial sequencer,  and should be managed through `x/sequencer` module.

Reply:
resolved?